setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = chemdyg
-version = 0.1.0
+version = 0.1.1
```

Comment:
Please try to remember to update this before making the release. Otherwise, there can be confusion about which version of the package is in a given release -- if you ask the code what version it is, it would say 0.1.0 even though the package will be created with version 0.1.1.  In this case, I can make a patch when we build the 0.1.1 release to fix this.

Reply:
Thank you, @xylar , for catching this.

Pinging @hsiangheleellnl . Perhaps when you complete the "area" changes, we can release version 0.1.2 and update the setup.cfg with the correct version number.

Reply:
Yes, if you make a 0.1.2 release, please do make sure to bump the version.  But I can work with 0.1.1 for now (with a patch to fix the release number).